### PR TITLE
Remove jQuery from interactor and components

### PR DESCRIPTION
**Pull request by FabHof**
This solves #219

### CODE REVIEW

**Comment by efx** [comment]
@FabHof thank you for making this improvement! Could you fix the linting error noted by the [CI failure](https://travis-ci.com/adopted-ember-addons/ember-pikaday/jobs/195896772#L499)?

**Comment by FabHof** [comment]
Linting error fixed

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-pikaday",
-  "version": "2.3.1",
+  "version": "2.3.2-lessjQuery",
```
Comment (efx) [inline package.json]: Could you remove this change please?

**Comment by efx** [comment]
I think we can remove this line as well: https://github.com/adopted-ember-addons/ember-pikaday/blob/master/package.json#L26

File: package.json (hunk 1 above)
Comment (FabHof) [reply package.json]: That should not have been pushed. sorry

Comment (efx) [reply package.json]: No problem!

File: addon/components/pikaday-input.js [hunk 2]
```diff
@@ -32,7 +32,10 @@ export default Component.extend(PikadayMixin, {
   },
 
   onPikadayClose() {
-    if (this.get('pikaday').getDate() === null || isEmpty(this.$().val())) {
+    if (
+      this.get('pikaday').getDate() === null ||
+      isEmpty(this.get('element.value'))
```
Comment (lifeart) [inline addon/components/pikaday-input.js]: this.elment = dom element,  don't think we can access dom properties using ember getter.
`this.get('element').value`

**Comment by efx** [comment]
@FabHof give we modified the `package.json`, I think we have to rebuild the `yarn.lock`. Could you run `yarn install --update-lock` and commit the `yarn.lock` changes?
From there, we will merge and release a minor version. Thank you again for making this addon svelte!

**Pull request merged**